tools/csvstoskyui/README.md

```diff
@@ -0,0 +1,17 @@
+# skynet
+
+## Tool to visualize topology from CSVs Using Modern Skydive UI
+- - - -
+
+![](https://github.com/skydive-project/skydive-ui/blob/master/tools/csvstoskyui/images/readme_screenshot.jpg?raw=true)
```

Comment:
Better if you display all edges created

You are missing Layers (all within 'Not Classified')

Reply:
I would also like a screenshot of the data center example present

Reply:
Look on the new version with scale ... (new PR with fixes)

Reply:
fixed (moved to scale view)